exercises/pangram/package.yaml

```diff
@@ -1,5 +1,5 @@
 name: pangram
-version: 1.0.0.2
+version: 1.0.0.3
```

Comment:
first three digits signify x-common version we are following. Since we are following 1.1.0, please 1.1.0.3 not 1.0.0.3

I am sorry that we did not document this (https://github.com/exercism/haskell/issues/538)